packages/common/src/HardhatConfig.js

```diff
@@ -0,0 +1,234 @@
+function getHardhatConfig(configOverrides) {
```

Comment:
This file is mostly the same as the previous buidler config. Main changes are updating from buidler to hardhat and all the associated import changes thereof.